docs/modules/user-guide/partials/assembly_migrating-to-devfile-v2.adoc

```diff
@@ -24,10 +24,11 @@ This guide explains how to migrate an existing v1.0 devfile to v2.0.
 [role="_additional-resources"]
 .Additional resources
 
-* New metadata: `version`. See link:https://github.com/che-incubator/devworkspace-api/issues/10[corresponding issue].
-* New mandatory metadata for plugins. See link:https://github.com/devfile/api/issues/31[corresponding issue].
+For a deeper understand of migrating devfiles, go to the following GitHub issues:
```

Comment:
I think you meant "deeper understanding". ;)

Honestly, I'd also say something like "for more information". We can't guarantee that someone will get a deeper understanding, but we can certainly link them to more details!

Reply:
good point. I addressed this concern in all the docs. resolving this conversation now!